frontend/src/lib/components/Spinner/Spinner.scss

```diff
@@ -1,18 +1,6 @@
 .Spinner {
     animation: spinner 2s infinite linear;
-    display: block;
```

Comment:
Shouldn't it be a block? For instance it's a bit misaligned in the "Monocolor" story when it's `inline`

Reply:
ah good catch. I removed it to see what effect it had but didn't commit me putting it back in